### ISSUE TITLE
Incorrect encoding

### ISSUE THREAD

**Issue by ghost** [5e-bits/5e-database#6]
Hi

Just been helping a friend debug some encoding issue with the API. Namely, the example we were on was for https://github.com/adrpadua/5e-database/blob/master/5e-SRD-Spells.json#L18.

Seems Windows-1252 has been stored as UTF-8 or vice versa (not quite sure which way around it is).

Solution for us was to do the following.

```python
import requests

res = requests.get('http://www.dnd5eapi.co/api/spells/1')
data = res.json()

print(data['material'])
>>> 'Powdered rhubarb leaf and an adderâ€™s stomach.'

data['material'] = data['material'].encode('windows-1252').decode('utf-8')
print(data['material'])
>>> 'Powdered rhubarb leaf and an adder’s stomach.'
```

The HTTP response headers are being sent in UTF-8 format, which is why I reckon this is being sent wrong, rather than being corrected by the library when it detects the encoding.

Hope this is of help.

**Comment by bobifle** [comment]
I have the same issue, seems to have been introduced by the commit 8dacafdfd2642f66e0c15c6edcc18cea2f3a0822 in  5e-SRD-Spells.json

**Comment by gabenunez** [comment]
@neko404notfound Thanks for the direction on this! 

I'm currently running a discord.js bot in node.js and had to install the UTF-8 and windows1252 modules to get this to work.

`item = utf8.decode(windows1252.encode(item));`

**Comment by justingolden21** [comment]
Here's what I've been doing (replacing strings):

<code>
itemDescription = itemDescription.split('â€™').join('\''); //fix apostrophie
itemDescription = itemDescription.split(',-').join('<br>').split('.,').join('.<br>');
</code>

Hopefully this will be useful for someone.

**Comment by bagelbits** [comment]
I think most of these should be fixed from #50, which just contains the fix from [this forked PR](https://github.com/bagelbits/5e-database/pull/1).

**Comment by bagelbits** [comment]
If you find anymore feel free to reopen the issue.
